doc/source/whatsnew/v0.20.0.txt

```diff
@@ -50,6 +50,7 @@ Other enhancements
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
 
+- pd.cut and qcut now support datetime64 and timedelta64 dtypes (issue:`14714`)
```

Comment:
made the changes now, just curious, is there a reason behind the double ticks? Does it get parsed by some code?